docs/source/user-guide/nlp/nlp_metadata.rst

```diff
@@ -0,0 +1,80 @@
+.. _nlp_metadata_guide:
+
+=================
+NLP Metadata
+=================
+
+Metadata is any additional information about the texts, that can be used to better understand the data and the model's
+behavior. As opposed to the unstructured text, metadata is structured, and is expected to be a pandas DataFrame.
+Metadata can be anything - from the source of the data, the timestamp of its creation, to the age of the author, and
+any other information that is given about the text.
+
+What Is Metadata Used For?
+=============================
+
+Metadata is used by Deepchecks' checks to extract meaningful features from the data, since some computations cannot be
+computed directly on the text (for example, drift). Inspecting the distribution of a metadata column's values (e.g. to
```

Comment:
cannot be computed directly on the text - not so clear what is the relation between that fact and the fact we ask for metadata. 
Would perhaps give weak segments as the first example here, as it's clearer and gives a better sense of "why should I care"

Reply:
yeah, it remained from properties accidentally. I changed the sentence a bit, tell me WDYT